tests/func/test_get.py

```diff
@@ -101,7 +101,6 @@ def test_get_from_non_dvc_master(erepo, tmp_path, monkeypatch, caplog):
     with caplog.at_level(logging.INFO, logger="dvc"):
         Repo.get(erepo._root_dir, erepo.FOO, out=imported_file, rev="branch")
 
-    assert caplog.text == ""
```

Comment:
Ok, so this definitely cannot be left like that:
1. if you are not checking caplog text, I would just remove caplog from this test.
2. In my opinion we have a problem here, that is probably not related to this issue:
- now every time someone will try to import from local with no cache, there will be information logged
`Setting 'upstream' to default`, which will definitely be confusing, as user did not set any remote during operation, and also, this operation is performed for temporary repo, and that will be even more confusing, as nothing in original repo will change, while information will be printed. @dmpetrov, do you agree with me?
What I think we should do is comment it out with `FIXME` comment above, and create issue on supressing at least some messages from `external_repo`. @efiop?

Reply:
@pared Good catch! For now, we could move that logging message from `RemoteConfig.add` to `dvc/command/remote.py:RemoteAdd` so it only shows up during CLI, as it was accidentaly moved from there during the last refactoring. Don't see any other place where we need to show that message, so seems like a good solution to me. What do you think?

Reply:
@efiop sounds good to me, only other way that I see would be to somehow supress logging in external repo. And just by saying it out loud I feel the cringe :D

Reply:
@maykulkarni I guess the consensus here is that this check should stay, we need to move the logging from `dvc/remote/config.py` to `CmdRemoteAdd`.